### DISCUSSION TITLE
Homebridge plugin with MiraI for Panasonic AC

### DISCUSSION THREAD

**Discussion by lnrath** [community/community#54123] [Programming Help]
### Select Topic Area

General

### Body

Can someone write a plugin for homebridge to integrate Panasonic AC with Homekit using Mirai login.

@embee8 wrote this https://github.com/embee8/homebridge-panasonic-ac-platform 

but it uses comfort cloud app login which is not available in India.

**Comment by thisisharishr** [comment]
Your are in luck, I just finished writing it. https://www.npmjs.com/package/homebridge-panasonic-miraie-ac-platform

**Reply by ccs777** [reply]
Thanks for this plugin. Can you add support for Fan mode.

**Comment by lnrath** [comment]
Thanks a lot. Was able to add it successfully...
